packages/deployment/app/cloudformation.template.yml

```diff
@@ -412,6 +412,10 @@ Resources:
         S3Bucket: {{this.bucket}}
         S3Key: {{this.remote}}
       FunctionName: {{../prefix}}-{{@key}}
+    {{# if this.layers}}
```

Comment:
Can we try updating this to handle one to many layers? Probably like so:

```
{{# if this.layers}}
  Layers:
  {{#each this.layers}}
    - {{this.layers}}
  {{/each}}
{{/if}}
```

Then your `lambdas.yml` entry would look like something like:

```
Lambda1:
  layers:
    - layer1
    - layer2
```

Reply:
@markdboyd I agree with your comment

I think if `layers` looks like this:
```
layers: 
  - layer1arn
  - layer2arn
```

you'd want this in the template:
```
{{# if this.layers}}
  Layers:
  {{#each this.layers}}
    - {{this}}
  {{/each}}
{{/if}}
```
Swapping out `this.layers` for `this`

@garysieling Where would the actual lambda layers be defined? Do you have an example?